src/DerConverter/Asn/DerAsnType.cs

```diff
@@ -53,6 +53,7 @@ internal static DerAsnType Parse(Queue<byte> data)
                 case DerAsnTypeTag.Ia5tring: throw new NotImplementedException();
                 case DerAsnTypeTag.UnicodeString: throw new NotImplementedException();
                 case DerAsnTypeTag.Sequence: return new DerAsnSequence(typeData);
+                case DerAsnTypeTag.UtcTime: return new DerAsnUtcTime(typeData);
```

Comment:
While at it, move this line 2 up (between Ia5String and UnicodeString) to keep the same order as in the `DerAsnTypeTag` enum. (makes me sleep better 😄 )